<?xml version="1.0" encoding="utf-8" ?>
<launch>
    <arg name="frame_topic" default="/zed/zed_node/left_raw/image_raw_color"/>
    <arg name="events_topic" default="/celex5_mipi/display/binary_img"/>
    <arg name="store_dir" default="$(find celex5_calibration)/output/"/>

    <include file="$(find zed_wrapper)/launch/zed.launch"/>
    <include file="$(find celex5_ros)/launch/celex5_ros_node.launch"/>

    <node pkg="celex5_calibration"
          name="extrinsics_calibration_node"
          type="extrinsics_calibration_node"
          output="screen"
          required="true">
        <param name="frame_topic" type="string" value="$(arg frame_topic)"/>
        <param name="events_topic" type="string" value="$(arg events_topic)"/>
        <param name="store_dir" type="string" value="$(arg store_dir)"/>
    </node>
</launch>
